packages/ios_platform_images/lib/ios_platform_images.dart

```diff
@@ -138,6 +138,74 @@ class IosPlatformImages {
     return _FutureMemoryImage(bytesCompleter.future, scaleCompleter.future);
   }
 
+  /// Loads an image from the system.  The equivalent would be:
+  /// `[UIImage systemImageNamed:name]`.
+  ///
+  /// Throws an exception if the image can't be found.
+  ///
+  /// [colors] takes a list of colors to be applied to the corresponding layer
+  /// of the icon, assuming that layer exists. An icon can have up to 3 layers,
+  /// if a layer color is not specified it will take the color of the most recent
+  /// valid index. Defaults to an empty list (black).
+  ///
+  /// For more information see [https://developer.apple.com/documentation/uikit/uiimagesymbolconfiguration/3810054-configurationwithpalettecolors?language=objc]
+  ///
+  /// [preferMulticolor] overrides [colors] and asks iOS to provide its preset
```

Comment:
Does this mean if preferMulticolor is set, colors should never be provided? If so, add an assert that encodes that.

Reply:
Nothing breaks if the colors are specified, it just ignores them. Should I still add the assertion?

Reply:
I would add it since it is confusing if you add colors later (e.g. while refactoring) and nothing changes because it got silently ignored.